typings/is-semver.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'is-semver' {
```

Comment:
@CjHare Is it worth pulling in this `is-semver` package?
https://www.npmjs.com/package/is-semver
When it doesnt have Definitely Typed types available. Fwiw this type definition isn't visible in the test files. Started looking into why its not, but its not actually needed there afaik.

Reply:
Ah right, this is why `./typings' is involved.

Okay, I'll have a think about the semver. My current thoughts are we probably don't need to validate the input i.e. not enforce semver, but I'm not entirely convinced yet 🤷